src/codestudio.js

```diff
@@ -0,0 +1,100 @@
+let redact;
+
+const CODESTUDIO_RE = /^(\[code-studio\s*)(\d+)?-?(\d+)?\]/;
+const CODESTUDIO = "codestudio";
+
+/**
+ * This plugin renders the special "codestudio" syntax used by
+ * CurriculumBuilder for the code studio pullthrough.
```

Comment:
Why is this sometimes `codestudio` and other times `code-studio`?

Reply:
I'm trying to emulate the original plugin; it identified itself as `codestudio` but looked for the syntax `code-studio`.